src/app/app.js

```diff
@@ -0,0 +1,46 @@
+import express from 'express';
+import queryParser from './middlewares/cookie-parser';
+import cookieParser from './middlewares/query-parser';
+import { users } from './models/users';
+import { products } from './models/products';
+import { reviews } from './models/reviews';
+
+const app = express();
+
+app.use(cookieParser);
+app.use(queryParser);
+app.use(express.json());
+
+app.get('/api/users', function (request, response) {
```

Comment:
Please, read [ this article](https://www.terlici.com/2014/09/29/express-router.html) and after it reafctor the structure and approach of applying routing middleware. In case of any questions do not hesitate to ask